config/packages/monolog.yaml

```diff
@@ -123,19 +123,70 @@ when@test:
 when@preprod:
     monolog:
         channels:
-            - sms
+            - login
+            - personal_data
+            - user
+            - relay
+            - affiliation
             - api
+            - sms
         handlers:
-            sms:
+            main:
+                type: fingers_crossed
+                action_level: error
+                handler: nested
+                excluded_http_codes: [ 404, 405 ]
+                buffer_size: 50 # How many messages should be saved? Prevent memory leaks
+            nested:
+                type: rotating_file
+                path: "%kernel.logs_dir%/%kernel.environment%.log"
                 level: debug
-                type: stream
-                path: '%kernel.logs_dir%/activity_dev/sms.log'
-                channels: [ 'sms' ]
+                max_files: 10
+            console:
+                type: console
+                process_psr_3_messages: false
+                channels: [ "!event", "!doctrine" ]
+            activity:
+                level: debug
+                type: rotating_file
+                path: '%kernel.logs_dir%/activity_preprod/login/login.log'
+                channels: [ 'login' ]
+                max_files: 365
+            personal_data:
+                level: debug
+                type: rotating_file
+                path: '%kernel.logs_dir%/activity_preprod/personal_data/personal_data.log'
+                channels: [ 'personal_data' ]
+                max_files: 365
+            user:
+                level: debug
+                type: rotating_file
+                path: '%kernel.logs_dir%/activity_preprod/user/user.log'
+                channels: [ 'user' ]
+                max_files: 365
+            relay:
+                level: debug
+                type: rotating_file
+                path: '%kernel.logs_dir%/activity_preprod/relay/relay.log'
+                channels: [ 'relay' ]
```

Comment:
Maybe we sould declare this as global handlers in order to avoid redefining them in all envs ?